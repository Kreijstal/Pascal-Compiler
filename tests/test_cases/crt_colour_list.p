program crt_colour_list;

uses crt;

var
  counter: integer;

begin
  clrscr;
  for counter := 0 to 15 do
  begin
    textcolor(counter);
    writeln(counter);
  end;
end.
