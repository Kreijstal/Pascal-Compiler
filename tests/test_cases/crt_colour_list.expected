[2J[H[0;30m0
[0;34m1
[0;32m2
[0;36m3
[0;31m4
[0;35m5
[0;33m6
[0;37m7
[0;90m8
[0;94m9
[0;92m10
[0;96m11
[0;91m12
[0;95m13
[0;93m14
[0;97m15
